tests/.env

```diff
@@ -11,7 +11,9 @@ UNDEFINED_EXPAND_WITH_DEFINED_NESTED=${UNDEFINED_ENV_KEY:-${MACHINE:-default}}
 UNDEFINED_EXPAND_WITH_DEFAULT=${UNDEFINED_ENV_KEY:-default}
 UNDEFINED_EXPAND_WITH_DEFAULT_NESTED=${UNDEFINED_ENV_KEY:-${UNDEFINED_ENV_KEY_2:-default}}
 UNDEFINED_EXPAND_WITH_DEFAULT_NESTED_TWICE=${UNDEFINED_ENV_KEY:-${UNDEFINED_ENV_KEY_2${UNDEFINED_ENV_KEY_3:-default}}}
```

Comment:
This is the failing test case with my new logic, how is this supposed to work? Should `UNDEFINED_ENV_KEY_2` and `${UNDEFINED_ENV_KEY_3:-default}` be concatenated? Why?

Reply:
I can make this test pass by making my algorithm expand the variables from left to right but I'm not sure I like this because the `matchGroup` regex gets a bit too loose for my tastes and the result will not match with the Bash expansion logic.

What do you think @motdotla and @vantreeseba?

Reply:
https://github.com/motdotla/dotenv-expand/pull/39#discussion_r842874752

The test case had a typo, I fixed it.